<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1370</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Конструкторская спецификация</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="3">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string> Алфавитный список деталей</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QTableWidget" name="tableWidget_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="sortingEnabled">
            <bool>false</bool>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderHighlightSections">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>позиция</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>кол-во</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>примечание</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>обозначение</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>название</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>раздел спецификации</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Введите код изделия</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Найти спецификацию</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Спецификация в порядке возрастания кода раздела и  позиции</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QTableWidget" name="tableWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="sortingEnabled">
            <bool>false</bool>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderHighlightSections">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>позиция</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>кол-во</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>примечание</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>обозначение</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>название</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>раздел спецификации</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1370</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Добавить</string>
    </property>
    <addaction name="add_user_action"/>
    <addaction name="add_spec_action"/>
    <addaction name="add_spec_section_action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="add_user_action">
   <property name="text">
    <string>Пользователя</string>
   </property>
  </action>
  <action name="add_spec_action">
   <property name="text">
    <string>Спецификацию</string>
   </property>
  </action>
  <action name="add_spec_section_action">
   <property name="text">
    <string>Раздел спецификации</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
